src/test/java/teammates/test/cases/webapi/SearchStudentsActionTest.java

```diff
@@ -37,9 +37,38 @@ protected void testExecute() {
     }
 
     @Test
-    public void execute_notEnoughParameters_parameterFailure() {
+    public void execute_invalidParameters_parameterFailure() {
         loginAsAdmin();
         verifyHttpParameterFailure();
+
+        String[] notEnoughParams = new String[] {
+                Const.ParamsNames.SEARCH_KEY, "dummy",
+        };
+        verifyHttpParameterFailure(notEnoughParams);
+
+        String[] invalidEntityParams = new String[] {
+                Const.ParamsNames.SEARCH_KEY, "dummy",
+                Const.ParamsNames.ENTITY_TYPE, "dummy",
+        };
+        verifyHttpParameterFailure(invalidEntityParams);
+    }
+
+    @Test
+    public void execute_wrongEntityType_parameterFailure() {
```

Comment:
This needs not be a separate test case as `wrongEntityType` is a subset of `invalidParameters`